grakn-graql-shell/src/main/java/ai/grakn/graql/GraqlShell.java

```diff
@@ -234,17 +234,8 @@ public static boolean runShell(String[] args, String version, String historyFile
 
         if (cmd.hasOption("b")) {
             try {
-                Optional<Integer> activeTasks = Optional.empty();
```

Comment:
Since these options are gone I think you may need to update the graql shell docs.